### PR TITLE
RNMT-3424 Add Dark Mode support

### PR DESCRIPTION
**Pull request by EiyuuZack**
## Description

Adds support for Dark to devices running iOS 13 and above. It also adds a few small features and fixes, such as the ability to manually enter the cardholder name, due to the bump from version 5.2.2 of the CardIO library.

## Context

Fixes https://outsystemsrd.atlassian.net/browse/RNMT-3424

The CardIO sample app could be updated to add this new field but it was out of the scope of this issue. See the commit message for more information.

## Type of changes

- [x] Feature (change which adds functionality)
    - [ ] BREAKING CHANGE (existing functionality will not work as expected due to new feature)
- [x] Fix (change which fixes an issue)
    - [ ] BREAKING CHANGE (existing functionality will not work as expected due to fix)
- [ ] Performance (change which improves performance)
    - [ ] BREAKING CHANGE (existing functionality will not work as expected due to performance improvement)
- [ ] Refactor (non-breaking change that is neither feature, fix nor performance)
- [ ] Style (non-breaking change that only affects formatting and/or white-space)

## Components affected
- [ ] Android
- [x] iOS
- [ ] JavaScript

## Tests

Manual tests

## Screenshots (if appropriate)

cardio_plugin

## Checklist

- [x] Pull request title follows the format `RNMT-XXXX <title>`
- [ ] Tests have been created
- [x] Code follows code style of this project
- [x] CHANGELOG.md file is correctly updated
- [ ] Changes require an update to the documentation
	- [ ] Documentation has been updated accordingly

### CODE REVIEW

File: src/ios/CardIO/CardIO.m [hunk 1]
```diff
@@ -0,0 +1,7 @@
+//
+// This file exists so that CocoaPods will generate a modulemap for CardIO
+//
+// See https://github.com/card-io/card.io-iOS-SDK/issues/115
+// and https://github.com/card-io/card.io-iOS-SDK/pull/126
+// for more details
```
Comment (Chuckytuh) [inline src/ios/CardIO/CardIO.m:6]: 👍

**Review by Chuckytuh — APPROVED** [review]

**Review by cmfsotelo — APPROVED** [review]
Quick question, how you managed to overcome the 100MB limit triggered by the binary file?
What was the resolution for that issue?

**Review by luissilvaos — APPROVED** [review]
Just for curiosity and I know the context of this issue was iOS only but, is the plugin working well with the dark mode in Android?

**Comment by EiyuuZack** [comment]
> Quick question, how you managed to overcome the 100MB limit triggered by the binary file?
> What was the resolution for that issue?

See the last commit of https://github.com/OutSystems/card.io-iOS-source/pull/1

**Comment by EiyuuZack** [comment]
> Just for curiosity and I know the context of this issue was iOS only but, is the plugin working well with the dark mode in Android?

Did not validate. However, Android is not as strict as iOS when it comes to Dark Mode:

- iOS is system wide **AND** imposed on the applications -- default text and background colours in apps will be changed by the system and can make apps look bad until fixed or they opt out of Dark Mode (temporary reprieve granted by Apple)
- Android is system wide **BUT NOT** imposed on the applications -- apps will be hinted the system is running in Dark Mode but the system will not do anything to the apps (hence there is no need for opting-out)

I hope this helps clear your question.

**Pull request merged**